src/components/Carousel.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Carousel = ({ images }) => (
+  <div
+    id="carouselExampleCaptions"
+    className="carousel slide relative"
+    data-bs-ride="carousel"
+  >
+    <div className="carousel-indicators absolute right-0 bottom-0 left-0 flex justify-center p-0 mb-4">
+      {images.map((image, index) => {
```

Comment:
- Sometimes, when the application takes a few seconds to load the images, the element just have a bad form (just for a few seconds), I consider using a default image while this load it will improve the UI.

<img width="828" alt="image" src="https://user-images.githubusercontent.com/20563252/179891643-cd495ad9-3e0b-4b9a-af13-343c8e604fbc.png">